3D/README.md

```diff
@@ -0,0 +1,25 @@
+# IWD 3D developer challenge
+
+## Setup
+
+Unity3D, a C# editor and few 3d meshes of your choice.
+
+## Guidelines
+
+We want to see how you tackle a technical problem.
+
+## Content
+
+The exercise consists in developing a camera focus on objects in the scene. Let's consider in that problem that we have a single camera in the scene using the perspective projection mode and you placed some items in the scene (whatever items you want).
```

Comment:
`the perspective projection mode` un lien vers la doc de Unity pour qu'il n'y ait pas de débat?